modules/log-upgrade/6x-uninstall-es-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-es-operator_{context}"]
+= Uninstalling Elasticsearch
+
+You can unintall Elasticsearch by using the {product-title} web console.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the *Administrator* perspective of the {product-title} web console.
+* If you have not already removed the {clo} and related resources, you must remove references to Elasticsearch from the `ClusterLogging` custom resource.
+
+.Procedure
+
+. Go to the *Administration* -> *Custom Resource Definitions* page, and click *Elasticsearch*.
+
+. On the *Custom Resource Definition Details* page, click *Instances*.
+
+. Click the Options menu {kebab} next to the instance, and then click *Delete Elasticsearch*.
+
+. Go to the *Administration* -> *Custom Resource Definitions* page.
+
+. Click the Options menu {kebab} next to *Elasticsearch*, and select *Delete Custom Resource Definition*.
+
+. Delete the object storage secret.
```

Comment:
Elasticsearch has not "object storage secret"

Reply:
removed